src/main/java/net/imagej/ui/swing/updater/ImageJUpdater.java

```diff
@@ -45,17 +46,9 @@
 import java.util.List;
 
 import net.imagej.ui.swing.updater.ViewOptions.Option;
+import net.imagej.updater.*;
```

Comment:
In ImageJ/SciJava, we avoid using wildcard imports, because it can break your code at runtime (see https://stackoverflow.com/a/150315/1919049 and http://www.javadude.com/posts/20040522-import-on-demand-is-evil/). (Maybe that's not true any more for Java 9+, where packages can't be cluttered in different jar files??).

I couldn't find this documented on the wiki (e.g. in https://imagej.net/Coding_style), but would vote for keeping explicit imports, even if it makes the header more verbose.